pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -620,10 +620,14 @@ public String getCrypterClassNameFromTableConfig(String tableNameWithType) {
     // Fetch the segment lineage metadata, and filter segments based on segment lineage.
     ZNRecord segmentLineageZNRecord =
         SegmentLineageAccessHelper.getSegmentLineageZNRecord(_propertyStore, tableNameWithType);
```

Comment:
Can we reuse `SegmentLineageAccessHelper.getSegmentLineage()` instead?

Reply:
Updated.